package.json

```diff
@@ -236,6 +229,7 @@
     "grunt-contrib-copy": "^1.0.0",
     "grunt-contrib-watch": "^1.1.0",
     "grunt-http": "^2.3.3",
+    "husky": "^9.1.7",
```

Comment:
errant caret?

Reply:
no – for this case, husky broke bigly and pinning was troublesome.  

The current version of husky loads config via distinct pre-commit, post-checkout, and post-merge files.  It was easy enough to modernize the package and safe to keep the caret.